mcs/class/corlib/Makefile

```diff
@@ -19,6 +20,7 @@ else
 $(error Unknown framework version)
 endif
 endif
+endif
```

Comment:
Won't we need the original --runtime flag from https://github.com/ntherning/mono/commit/3891a8907626f741037ec76a282c4a89c65aef46#diff-2d1c64decd91d9a6e8842ab0f0e9438dL14 to make it work for mcs?